english/I'm-batman/catwoman/catwoman.md

```diff
@@ -0,0 +1,3 @@
+The batphone rings. Its commissioner Gordon telling you Catwoman just robbed Gotham National Bank!
+
+Do you pursue Catwoman? [Yes](../pursue/pursue.md) or [No](../vacation/vacation.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.